### ISSUE TITLE
Exception when viewing external drive root

### ISSUE THREAD

**Issue by Arlodotexe** [duke7553/files-uwp#297]
**Describe the bug**
When navigating to the root of an external drive, it throws an ArgumentException

**To Reproduce**
1. Open the app
2. Select an external drive from the sidebar
3. Error

**Expected behavior**
No error?

**Screenshots**
Here's the exact line causing the bug (In `InstanceTabsView.xaml.cs`):
![image](https://user-images.githubusercontent.com/9384894/68825653-810d9700-0660-11ea-8ead-3a285ababb58.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro 1909 (18363.449)
 - App version: 0.6.5.0

**Additional context**
Building from source, as release x64 in VS 2019 Community (latest update)

**Comment by SOI7** [comment]
I confirm this issue, it's happening to me as well with my secondary hard drive. It didn't happen with the previous version

**Comment by yaichenbaum** [comment]
There were some changed made to how external drives work in this update, its is very likely this is what is causing this issue. Thank you for reporting it.

**Comment by duke7553** [comment]
Sorry everybody for the inconvenience. I will investigate this issue this afternoon and we'll release a patch.

**Comment by generalguy41** [comment]
Assuming this is the same error, I don't think it's limited to external drives. I have a network drive and an internal drive that give the same error. I also have an external drive and USB that don't give the error. Downloaded from Store.

```
Friendly Description:
   Value does not fall within the expected range. within method (Method name unknown)
Technical Information:
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task) + 0x70
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task) + 0x38
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task) + 0x17
   at Files.InstanceTabsView.<SetSelectedTabInfo>d__5.MoveNext() + 0x418
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__7_0(Object) + 0x1e
   at System.Action`1.Invoke(T) + 0x28
   at System.Threading.WinRTSynchronizationContext.Invoker.InvokeCore() + 0x33
   at System.Runtime.InteropServices.McgMarshal.ThrowOnExternalCallFailed(Int32, RuntimeTypeHandle) + 0x21
   at __Interop.ComCallHelpers.Call(__ComObject, RuntimeTypeHandle, Int32) + 0xb8
   at __Interop.ForwardComStubs.Stub_12[TThis](__ComObject, Int32) + 0x24
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object, UnhandledErrorDetectedEventArgs) + 0x3d
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw() + 0x21
   at Microsoft.AppCenter.Utils.ApplicationLifecycleHelper.<.ctor>b__17_1(Object, UnhandledErrorDetectedEventArgs) + 0x75
   at System.EventHandler`1.Invoke(Object, TEventArgs) + 0x2e
   at __Interop.Intrinsics.HasThisCall__24[TArg0](Object, IntPtr, Object, TArg0) + 0x36
   at Files!<BaseAddress>+0xec2150
```

**Comment by duke7553** [comment]
I have a fix for this locally, and I'm working on several other fixes for the v0.6.6 release which will be released soon.
